src/test/scala/edu/ie3/mobsim/model/ChargingBehaviorSpec.scala

```diff
@@ -368,5 +369,33 @@ class ChargingBehaviorSpec extends UnitSpec with ChargingBehaviorTestData {
         UUID.fromString("7537c0b6-3137-4e30-8a95-db1c0f9d9b81") -> 5.0
       )
     }
+
+    "return the correct price rating" in {
+      val priceRating = PrivateMethod[Double](Symbol("priceRating"))
+
+      val evWithLowPriceMemory: ElectricVehicle = ev1
+      evWithLowPriceMemory.updateChargingPricesMemory(0.0)
+
+      val evWithHighPriceMemory: ElectricVehicle = ev1
+      evWithHighPriceMemory.updateChargingPricesMemory(0.0)
+      evWithHighPriceMemory.updateChargingPricesMemory(1.0)
+
+      val cases = Table(
+        ("chargingStation", "ev", "prices", "resultingRating"),
+        (cs6, ev1, Map.empty[UUID, Double], 1.0),
+        (cs6, ev1, Map(cs6.getUuid -> 0.0), 1.0),
+        (cs6, evWithLowPriceMemory, Map(cs6.getUuid -> 0.0), 0.5),
+        (cs6, evWithHighPriceMemory, Map(cs6.getUuid -> 0.0), 1.0)
+      )
```

Comment:
The `chargingStation` parameter is obsolete here, since it's just `cs6` for all rows